tools/explorer/app.py

```diff
@@ -42,23 +63,59 @@ def main(parsed_args):
       for field in fields:
         print(field)
     elif function_choice == '2':
-      print('This functionality is still under construction.\n' +
-            'Please pick another option.')
+
+      standard_field_list = _InputFields(my_ontology)
+
+      entity_type_match_dict = {}
+      for i, match in enumerate(my_ontology.GetEntityTiypesFromFields(
+          standard_field_list
+      )):
+        entity_type_match_dict[i] = match
+      print('How many matches would you like to see?')
```

Comment:
question: should we have a default option ?

Reply:
I made it so the top ten matches are shown initially and the user can choose to see all matches.